integreat_cms/cms/models/users/organization.py

```diff
@@ -69,7 +69,7 @@ def num_pages(self) -> int:
 
         :return: the current number of maintained pages of an organization object
         """
-        return self.pages.count()
+        return self.pages.count() + self.pois.count()
```

Comment:
This executes two database queries and adds the resulting integers in python. I went a little crazy and wrote a utility function for getting the count of multiple querysets as a single db query.

```suggestion
        return combined_count(self.pages, self.pois)
```

some_utils.py:
```python
def combined_count(*args: *Queryset) -> int:
    """
    Efficiently calculate the combined count of the given querysets on the database.
    This runs a single query, as opposed to ``A.objects.count() + B.objects.count()``
    which results in two queries and does the adding in python.
    """
    if not args:
        return 0
    if len(args) == 1:
        full = args[0]
    else:
        # Since we are only interested in the count, and the querysets might be from different models,
        # only selects the primary keys and reset any previous sorting using order_by without arguments
        querysets = [q.values("pk").order_by() for q in args]
        # PKs may be from different models, so we want to allow duplicates → all=True
        full = querysets[0].union(*querysets[1:], all=True)
    return full.count()
```

This might be a bit overkill since the only other use I found with a quick simple grep is `integreat_cms/cms/models/media/directory.py:68`, but who knows. I was curious about this problem, why throw away what I learned?

Reply:
Tbh, I would be a bit hesitant towards adding this, since it increases the complexity of the code without really having a performance improvement (sub 5ms on a region with 55 organization, burgenlandkreis). I guess what we could do if we really wanted to improve the performance of the region list would be to use `annotate` on the organizations query, to load the counts for every organization in one query? Though that should probably be its own issue.

Reply:
I suggest to do it as a separate issue too (if we do)🤔